app/views/active_admin/resource/_page_event_component_form.html.arb

```diff
@@ -36,6 +36,14 @@ html = Arbre::Context.new do
           para 'URL used for event title link, e.g. https://va.gov', class: 'inline-hints'
         end
 
+        li class: 'string input optional stringish', id: "page_page_components_attributes_#{placeholder}_component_attributes_presented_by_input" do
+          label 'Presented by', for: "page_page_components_attributes_event_#{placeholder}_component_attributes_text", class: 'label'
```

Comment:
I think I may be a bit confused but should this say:

`for: "page_page_components_attributes_event_#{placeholder}_component_attributes_presented_by"`

It seems like some of the `li`s follow this pattern and maybe a few of them have `text` in there by mistake?

Reply:
👀 Ooh, good catch. I'll take another look